app/views/dit_landing_page/_training.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="landing-page__section dit-landing-page__section--dark-grey landing-page__section--border-bottom-red">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-heading-l">
+          <%= t("dit_landing_page.training_section_title") %>
+        </h2>
+        <%= render "govuk_publishing_components/components/govspeak", {
+        } do %>
+          <%= t("dit_landing_page.training_section_description").html_safe() %>
+        <% end %>
+        <%= render "govuk_publishing_components/components/table", {
+          head: [
```

Comment:
We could pull this in from the translation file

Reply:
Thanks, done 👍